src/scss/global.scss

```diff
@@ -6,6 +6,7 @@
 @import "node_modules/bootstrap/scss/bootstrap";
 @import "node_modules/bootswatch/dist/darkly/bootswatch";
 @import "./constants.scss";
+@import "./ClientCustom/Custom.scss";
```

Comment:
I have mixed feels about this. We’re generally trying to eradicate things from the global scss.. It’s probably not the worst thing.. it’s better than hacking in from the DB.

Reply:
Precisely, a compromise